PluralKit.Bot/Commands/Groups.cs

```diff
@@ -224,6 +225,53 @@ async Task ShowIcon()
             else
                 await ShowIcon();
         }
+        public async Task GroupColor(Context ctx, PKGroup target)
+        {
+            var color = ctx.RemainderOrNull();
+            if (await ctx.MatchClear())
+            {
+                ctx.CheckOwnGroup(target);
+                
+                var patch = new GroupPatch {Color = Partial<string>.Null()};
+                await _db.Execute(conn => _repo.UpdateGroup(conn, target.Id, patch));
+                
+                await ctx.Reply($"{Emojis.Success} Group color cleared.");
+            }
+            else if (!ctx.HasNext())
+            {
+
+                if (target.Color == null)
+                    if (ctx.System?.Id == target.System)
+                        await ctx.Reply(
+                            $"This group does not have a color set. To set one, type `pk;group {target.Reference()} color <color>`.");
+                    else
+                        await ctx.Reply("This group does not have a color set.");
+                else
+                    await ctx.Reply(embed: new EmbedBuilder()
+                        .Title("Group color")
+                        .Color(target.Color.ToDiscordColor())
+                        .Thumbnail(new($"https://fakeimg.pl/256x256/{target.Color}/?text=%20"))
+                        .Description($"This group's color is **#{target.Color}**."
+                                         + (ctx.System?.Id == target.System ? $" To clear it, type `pk;group {target.Reference()} color -clear`." : ""))
+                        .Build());
+            }
+            else
+            {
+                ctx.CheckOwnGroup(target);
+
+                if (color.StartsWith("#")) color = color.Substring(1);
```

Comment:
I think we have a utility function for color parsing like this?

Reply:
If we do, it's not used in the member color command either! I just mostly referenced/copied over the code from the member command